package.json

```diff
@@ -23,6 +23,8 @@
     "format:graphql": "prettier --write --parser graphql '**/*.graphql'",
     "start": "yarn playground:start",
     "prerelease": "yarn auth && yarn build && yarn typecheck",
+    "prerelease:alpha": "yarn prerelease",
```

Comment:
Will this be published to another dist-tag? Should it maybe while also having another prefix in the name of the bundle?

Reply:
Hmm I'm not sure I follow 🧐

Reply:
Ah sorry, gotta get some coffee. Didn't see the `pre` infront.